freezeyt/url_finders.py

```diff
@@ -0,0 +1,60 @@
+import xml.dom.minidom
+from typing import List
+from urllib.parse import urljoin
+
+import html5lib
+import css_parser
+
+from werkzeug.datastructures import Headers
+from werkzeug.http import parse_options_header
+
+from freezeyt.encoding import decode_input_path
+
+
+def get_css_links(
+    css_file: bytes, base_url: str, headers: Headers = None
+)  -> List[str]:
```

Comment:
* Dal bych `-> Iterable[str]`; tuhle funkci kdykoli můžeme třeba změnit na generátor.

Reply:
jasně díky na to jsem zapomněl "obecnější"